### PR TITLE
Ignoring empty files

### LINKED ISSUE #1: Ignoring empty files like __init__.py
**Is your feature request related to a problem? Please describe.**
Ignoring the empty files, like `__init__.py`

**Describe the solution you'd like**
Add a configuration option `file-header-ignore-empty` to ignore if file is empty

**Describe alternatives you've considered**
I didn't find a good solution to ignore only empty files in pylint, adding all the files to ignore is too much of work

### PR DESCRIPTION
**Pull request by SVilgelm**
Add `file-header-ignore-empty-files` configuration option to ignore empty files

closes #1

### CODE REVIEW

**Comment by SVilgelm** [comment]
Hi @HaaLeo, could you please review this pull request. The Travis CI fails due to unrelated issues in python 3.4:
```
Collecting Pygments>=2.5.1 (from readme-renderer>=21.0->twine>=1.11.0->-r requirements-dev.txt (line 12))
268  Could not find a version that satisfies the requirement Pygments>=2.5.1 (from readme-renderer>=21.0->twine>=1.11.0->-r requirements-dev.txt (line 12)) (from versions: 0.5, 0.5.1, 0.6, 0.7, 0.7.1, 0.8, 0.8.1, 0.9, 0.10, 0.11, 0.11.1, 1.0, 1.1, 1.1.1, 1.2, 1.2.1, 1.2.2, 1.3, 1.3.1, 1.4, 1.5, 1.6rc1, 1.6, 2.0rc1, 2.0, 2.0.1, 2.0.2, 2.1, 2.1.1, 2.1.2, 2.1.3, 2.2.0, 2.3.0, 2.3.1)
269No matching distribution found for Pygments>=2.5.1 (from readme-renderer>=21.0->twine>=1.11.0->-r requirements-dev.txt (line 12))
```

File: .travis.yml [hunk 1]
```diff
@@ -17,12 +17,10 @@ script:
 
 jobs:
   include:
-      # Test Stage
+    # Test Stage
     - stage: test
       python: 2.7
       script: pytest --cov pylintfileheader pylintfileheadertest # Do not lint here
-    - stage: test
```
Comment (HaaLeo) [inline .travis.yml]: I would prefer to keep the python 3.4 CI build. Do you have an idea how we could retain the python 3.4 build?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # pylint-file-header
 
-[![Pypi](https://img.shields.io/pypi/v/pylintfileheader.svg?style=flat-square)](https://pypi.python.org/pypi/pylintfileheader) [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/pylintfileheader.svg?style=flat-square)](https://pypi.python.org/pypi/pylintfileheader) [![PyPI - Downloads](https://img.shields.io/pypi/dm/pylintfileheader.svg?style=flat-square)](https://pypistats.org/packages/pylintfileheader) [![Stars](https://img.shields.io/github/stars/HaaLeo/pylint-file-header.svg?label=Stars&logo=github&style=flat-square)](https://github.com/HaaLeo/pylint-file-header/stargazers)
```
Comment (HaaLeo) [inline README.md]: Can you please add the trailing spaces for the badge files again. Then they are rendered with less space between them.

File: pylintfileheadertest/file_header_checker_test.py [hunk 3]
```diff
@@ -51,3 +51,23 @@ def test_config_empty_no_message_added(self):
         node_mock.stream.return_value.__enter__.return_value.read.return_value.decode.return_value = '# Invalid\n# Header'
         with self.assertNoMessages():
             self.checker.process_module(node_mock)
+
+    def test_ignore_empty_files(self):
```
Comment (HaaLeo) [inline pylintfileheadertest/file_header_checker_test.py:55]: Thx for adding tests 🚀

**Review by HaaLeo — CHANGES_REQUESTED** [review]
Hi @SVilgelm thx for your contribution. The major changes look good. I just requested a minor change. 

To be honest I would prefer to include the python 3.4 build before merging the PR. Do you have an idea for a fix or are willing to investigate the issue?

File: README.md (hunk 2 above)
Comment (SVilgelm) [reply README.md]: done, it was an auto-formatting

File: pylintfileheadertest/file_header_checker_test.py (hunk 3 above)
Comment (SVilgelm) [reply pylintfileheadertest/file_header_checker_test.py:55]: sure thing

File: .travis.yml (hunk 1 above)
Comment (SVilgelm) [reply .travis.yml]: working on it

Comment (SVilgelm) [reply .travis.yml]: Done by pinning the `readme-renderer` to `24.0` for python3.4

**Comment by SVilgelm** [comment]
@HaaLeo the PR is ready

**Review by HaaLeo — APPROVED** [review]

**Comment by SVilgelm** [comment]
thank you!

**Pull request merged**